entries/d/deutschepost.de.json

```diff
@@ -0,0 +1,19 @@
+{
+  "Deutsche Post": {
+    "domain": "deutschepost.de",
+    "additional-domains": [
+      "epost.de"
+    ],
+    "contact": {
+      "facebook": "deutschepost",
+      "language": "de"
+    },
+    "keywords": [
+      "postal",
+      "email"
```

Comment:
They do not actually offer an email service, even though they use "name@epost.de" style addresses. Users can send and receive documents electronically only within that system, or convert physical letters from/to electronic ones. Formerly, the entry was in `communication`, but I'm okay with using `postal` only.

Reply:
Oh, haha :sweat_smile: 
I'll change it